docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -11,10 +11,10 @@ lang: en
 
 - [Registering a protocol](#registering-a-protocol)
   - [Registering a protocol on Windows](#registering-a-protocol-on-windows)
-  - [Registering a protocol on OSX](#registering-a-protocol-on-osx)
+  - [Registering a protocol on macOS](#registering-a-protocol-on-macOS)
```

Comment:
Second though, the only difference I see between the two mac section is the `Then, delete the following lines in the info.plist file, which fall between the` part.

If this is the case, can we keep only one macOS section and add an extra step "Additional required changes in Info.plist for BigSur and newer" ?

This would avoid a duplication